Source/WTF/wtf/PrintStream.cpp

```diff
@@ -146,7 +146,7 @@ void printInternal(PrintStream& out, unsigned char value)
 
 void printInternal(PrintStream& out, char16_t value)
 {
-    out.printf("%lc", value);
+    out.printf("%lc", static_cast<wint_t>(value));
```

Comment:
Where'd this one come from?

Reply:
With printf format warning option, this becomes warning.